docs/Tutorials/Contracts/Account-Funds-Transfers.md

```diff
@@ -77,7 +77,7 @@ console.log("Account B has an updatedbalance of: " + accountBBalance);
 }
 ```
 
-A [full example](https://github.com/ConsenSys/quorum-dev-quickstart/blob/master/files/besu/smart_contracts/scripts/eth_tx.js)
+A [full example](https://github.com/ConsenSys/quorum-dev-quickstart/blob/b8f7e3bc1d5f773b823c5e9983e9da11bb552ed7/files/besu/smart_contracts/privacy/scripts/eth_tx.js)
```

Comment:
is it better to link to a specific commit? That would mean that if we ever change this js in the future, this link will be to an old version. 
it does still exist on master at 
https://github.com/ConsenSys/quorum-dev-quickstart/blob/master/files/besu/smart_contracts/privacy/scripts/eth_tx.js

Reply:
Yes, that's the point.
We need to point to specific commits because otherwise, if you change the file without notifying the doc team, we have the risk of this file not matching our docs and users not being able to run through the instructions.
It will also prevent 404 errors when the files are moved or deleted.
However, we are planning to add a weekly CI check that :
- all links to github files with a commit hash are up to date (no changes since this commit) (warning level message)
- all links to github files use the commit hash. (error level message)